CHANGELOG.md

```diff
@@ -4,6 +4,38 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [4.0.0.2] Unreleased
```

Comment:
You can leave the changelog. I have a local updated version which I will include before merging to master.

Reply:
I wont.

Reply:
But you can. I think it is appropriate to include changelog.

Reply:
I will overwrite it. It duplicates previous additions and it is not necessary to keep the changelog fine grained. We have a commit history for those who want to delve deeper.

Reply:
It's ok if you overwrite it. I think I removed overly detailed stuff from it, but whatever.